specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -9185,6 +9256,64 @@
         "modelAsString": true
       }
     },
+    "IntegrationServiceEnvironmentManagedApiListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApi"
+          },
+          "description": "The integration service environment managed APIs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "The list of integration service environment managed APIs."
+    },
+    "IntegrationServiceEnvironmentManagedApi": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApiProperties",
+          "description": "The integration service environment managed api properties."
+        }
+      },
+      "description": "The integration service environment managed api.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "IntegrationServiceEnvironmentManagedApiProperties": {
+      "type": "object",
+      "description": "The integration service environment managed api properties.",
+      "properties": {
+        "deploymentParameters": {
+          "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApiDeploymentParameters",
+          "description": "The integration service environment managed api deployment parameters."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ApiResourceProperties"
```

Comment:
May I ask whether the `ApiResourceProperties` under the `IntegrationServiceEnvironmentManagedApi` can be removed if we don't need it?

Reply:
None of the properties neeeds to be set while creating. But, if you dont set it, we will give default values.

Reply:
So you mean this property is read-only? Or it's optional and it can be set while creating/updating? If it's optional property, is there any sample or document talking about how to set these sub properties under ApiResourceProperties since it includes lots of sub properties?

Reply:
Actually apart from Integration Service Environment all the others are readonly and IntegrationServiceEnvironment is optional. Update this accordingly.